src/main/java/mods/eln/Eln.java

```diff
@@ -6640,6 +6678,29 @@ void recipemagnetiser() {
 				new ItemStack[] { findItemStack("Advanced Magnet") }, 15000.0));
 	}
 
+	void recipeFuelBurnerItem() {
+		addRecipe(findItemStack("Small Fuel Burner"),
+			"   ",
+			" Cc",
+			"   ",
+			Character.valueOf('C'), findItemStack("Combustion Chamber"),
+			Character.valueOf('c'), findItemStack("Copper Thermal Cable"));
+
+		addRecipe(findItemStack("Medium Fuel Burner"),
+			"   ",
+			" Cc",
+			"   ",
+			Character.valueOf('C'), findItemStack("Combustion Chamber", 2),
```

Comment:
This doesn't work; you can't require multiple chambers in the same slot. Try spreading them out.